thetis/options.py

```diff
@@ -888,16 +899,23 @@ def add_tracer_2d(self, label, name, filename, shortname=None, unit='-', source=
         assert label not in self.tracer, f"Field '{label}' already exists."
         assert ' ' not in label, "Labels cannot contain spaces"
         assert ' ' not in filename, "Filenames cannot contain spaces"
+        if priority is not None:
+            assert isinstance(priority, int)
+            if priority in self.tracer_priorities:
```

Comment:
Would it not be simpler to just ask the user to add the tracer in the order they want them solved rather than specifying a priority? Or is there a separate meaning to the order in `self.tracer` that we might want to be different from the solve order?